modules/api/api-xsjs/src/main/java/com/sap/xsk/api/destination/CloudPlatform.java

```diff
@@ -30,12 +30,10 @@
   @Override
   public JsonObject getServiceCredentials(@Nonnull String serviceName, @Nullable String servicePlan)
       throws CloudPlatformException {
-    String destinationPrefix = Configuration.get(KymaModule.DIRIGIBLE_DESTINATION_PREFIX, "");
-
-    String clientId = getWithPrefix(destinationPrefix, KymaModule.DIRIGIBLE_DESTINATION_CLIENT_ID);
-    String clientSecret = getWithPrefix(destinationPrefix, KymaModule.DIRIGIBLE_DESTINATION_CLIENT_SECRET);
-    String url = getWithPrefix(destinationPrefix, KymaModule.DIRIGIBLE_DESTINATION_URL);
-    String uri = getWithPrefix(destinationPrefix, KymaModule.DIRIGIBLE_DESTINATION_URI);
+    String clientId = Configuration.get(KymaModule.DIRIGIBLE_DESTINATION_CLIENT_ID);
+    String clientSecret = Configuration.get(KymaModule.DIRIGIBLE_DESTINATION_CLIENT_SECRET);
+    String url = Configuration.get(KymaModule.DIRIGIBLE_DESTINATION_URL);
+    String uri = Configuration.get(KymaModule.DIRIGIBLE_DESTINATION_URI);
```

Comment:
These variables are currently missing in the [KymaModule.java](https://github.com/eclipse/dirigible/blob/master/releng/sap-kyma-base/src/main/java/org/eclipse/dirigible/kyma/KymaModule.java)